source/fundamentals/connection/connect.txt

```diff
@@ -4,6 +4,14 @@
 Connection Guide
 ================
 
+.. default-domain:: mongodb
```

Comment:
Suggestion:
This directive doesn't do anything anymore, please remove when you see it.